app/src/lib/api.ts

```diff
@@ -57,13 +59,42 @@ export interface IAPIRepository {
  */
 export interface IAPICommit {
   readonly sha: string
-  readonly author: IAPIUser | null
+  readonly author: IAPIIdentity | {} | null
+}
+
+/**
+ * Entity returned by the `/user/orgs` endpoint.
+ *
+ * Because this is specific to one endpoint it omits the `type` member from
+ * `IAPIIdentity` that callers might expect.
+ */
+export interface IAPIOrganization {
+  readonly id: number
+  readonly url: string
+  readonly login: string
+  readonly avatar_url: string
+}
+
+/**
+ * Minimum subset of an identity returned by the GitHub API
+ */
+export interface IAPIIdentity {
+  readonly id: number
```

Comment:
**Non-blocking**

Is it possible to use a sum/intersection type here to take advantage of the fact that these same fields are defined in the type above?

Reply:
I didn't really see the value in coupling these together to save on typings because it felt like a different version of our mistake of assuming `IAPIUser` everywhere. Also, using `IAPIOrganization` (a special case returned by a specific API) to derive `IAPIIdentity` (a more general case used broadly in the API) doesn't make sense to me from a composition side.